src/gql/benefactor.d.ts

```diff
@@ -0,0 +1,24 @@
+export type BenefactorFormData = {
+  // id: string
+  extCustomerId: string
+  // createdAt: string
+  // updatedAt: string
+  person: string
+}
+
+export type BenefactorInput = {
+  // id: string
+  extCustomerId: string
+  // createdAt: string
+  // updatedAt: string
+  person: string
```

Comment:
What's the expected data that will go in the `person: string` field? is that the name of the person or something else

Normally in this field, we have the expanded person data like id, first name, last name, email, etc.